ntc_templates/templates/alcatel_sros_show_lag_port.textfsm

```diff
@@ -0,0 +1,21 @@
+Value LAG_ID (\d+\(\w\))
+Value PORT_ID (\d+/\d+/\d+)
+Value ADM (up|down)
+Value ACT_STD (active|standby)
+Value OPR (up|down)
+Value PRIMARY (yes|no)
+Value SUB_GROUP (\d+)
+Value FORCED (-)
+Value PRIO (\d+)
+
+Start
+  ^----------- -> LagPort
```

Comment:
```suggestion
Start
  ^-+ -> LagPort
  ^=+
  ^Lag\s+Port\s+States
  ^LACP\s+Status:
  ^Lag-id\s+
  ^\s+Stdby\s*
  ^\s*$$
  ^. -> Error
```

I'd reduce the characters for the horizontal separators by using a regex with a plus sign.

Also add in the extra lines (before the state transition to `LagPort`) and `Error` rule for the `Start` state.

Reply:
Not looking to rag on you -- simply inform.
https://github.com/networktocode/ntc-templates/blob/master/docs/user/faq.md#why-is-there-a-requirement-to-use-error-in-every-template
https://github.com/networktocode/ntc-templates/blob/master/docs/dev/dev_parser.md#error-directive

Precedent template with `Error` in each state.
https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/alcatel_sros_show_router_mpls_interface.textfsm

Reply:
Thanks, was not aware! I've incorporated these changes 👍